deploy/sdk/src/dynamo/sdk/tests/test_probes.py

```
@@ -0,0 +1,91 @@
+import subprocess
+import time
+
+import pytest
+from typer.testing import CliRunner
+
+pytestmark = pytest.mark.pre_merge
+runner = CliRunner()
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_and_teardown():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)